Sources/BraintreeCore/BTAPIPinnedCertificates.swift

```diff
@@ -8,152 +8,456 @@ import Foundation
 ///
 /// Do not update this file directly. If new certificates are needed, update the `certificates` directory
 /// in `braintree-ios-certificate-generation` and re-run the certificate generation script.
-class BTAPIPinnedCertificates {
+@objcMembers public class BTAPIPinnedCertificates: NSObject {
```

Comment:
In [this PR](https://github.com/braintree/braintree_ios/pull/1052/files#diff-e0d221245e7631e0b1447a0720aee07ff3ee3b517f87dd68aaf1c377ec8f4e41) I updated this class/method to be non-public but didn't actually update the script. 🤦‍♂️ I'll put up a small PR to the cert script to make this change, but we will likely need to run the script again and update this PR once merged.

Reply:
Ok no worries. Does your linked PR impact the merging of the new certs?

Reply:
Ok I'll pull the latest from main and run it again!